### DISCUSSION TITLE
text size

### DISCUSSION THREAD

**Discussion by Paull26** [david-allison/manx-search-data#1082] [General]
Can we make the size of text of footnotes smaller than the main text,?For example in Yn Tiger.

**Comment by david-allison** [comment]
Hi Paul,

I'm in Taiwan right now with limited free time, I'll be back on Island around the 20th.

This should be done in: https://github.com/david-allison/manx-corpus-search/pull/171

I haven't tested this, please let me know if it works

Best wishes,
David
